packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -21,6 +21,13 @@ enum CollapseMode {
 
   /// The background widget will act as normal with no collapsing effect.
   none,
+
```

Comment:
The first paragraph of documentation must be a short self-contained sentence that explains the purpose and meaning of the item being documented.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements